src/components/diff/utils.ts

```diff
@@ -0,0 +1,2 @@
+export const toUppercase = (str: string) => str.charAt(0).toUpperCase() + str.slice(1);
+export const toLowercase = (str: string) => str.charAt(0).toLowerCase() + str.slice(1);
```

Comment:
Let's have these live in the generic `src/lib/utils.ts` file since that file is small enough that we can still keep everything in one place, instead of having various utils files